### DISCUSSION TITLE
Common app/data model/theme

### DISCUSSION THREAD

**Discussion by PowerRobin** [microsoft/Power-Platform-For-Beginners#9] [General]
Hey everyone,

I think it would be really cool if we had a common topic for the whole beginner repo.

By that I mean a common core data model all the examples revolve around. My suggestion would be something really generic like "project management" that is relatable for everyone. 

Technically speaking we probably could hop together onto a dev tenant and have a core solution with the tables and probably a Model Driven App.
Every other chapter comes with its own solution and builds on the same data model.

I guess it's a little more complicated then everyone doing their own thing, but would be a much nicer experience in the end for the beginners always coming back to the same theme and truly learning about "better together"!

Please share your thoughts and ideas!

**Comment by fboucher** [comment]
I'll close this discussion now that most happens on Teams and keep only the more general ones open for now. :)

@PowerRobin did you get access to Teams? If not ping me!
